core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review-modal.component.ts

```diff
@@ -173,7 +173,12 @@ export class QuestionSuggestionReviewModalComponent
       modalRef.componentInstance.skill = skillDict.skill;
       modalRef.componentInstance.skillDifficulty = this.skillDifficulty;
 
-      modalRef.result.then(() => {
+      modalRef.result.then((change) => {
+        this.allContributions[this.suggestionId].suggestion.change
```

Comment:
Can you add a comment here explaining that we need to update the component's question suggestion data so that the changes are reflected in the UI?

Reply:
Done.